documentation/C++style.md

```diff
@@ -15,7 +17,15 @@ is clear on usage, follow it.
 * Reasonable exceptions to these guidelines can be made.
 * Be aware of some workarounds for known issues in older C++ compilers that should
   still be able to compile f18. They are listed at the end of this document.
+
 ## In particular:
+
+Use serial commas in comments, error messages, and documentation.
```

Comment:
... unless they introduce ambiguity?

I'm not sure that we want to say much about English usage in a C++ style guide, other than to encourage correct spelling and clear syntax.